### PR TITLE
See all reminders

### PR DESCRIPTION
**Pull request by Pilewski**
## Purpose

When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

Each reminder should have a class of .spec-reminder-item.
There is a failing test set up in tests/acceptance/reminder-list-test.js.

## Approach

- Run ember g route for index and reminders
- Set up index redirect to reminders route
- Set up reminder model
- Update reminders.hbs to map through and render reminders from model

Closes issue #1

### Learning

_Describe the research stage._

_Links to blog posts, patterns, libraries or add-ons used to solve this problem._

### Test coverage 

-N/A

### Follow-up tasks

- [Issue #2 (Example)](https://github.com/flexyford/pull-request/issues)

#### After

@brittanystoroz @martensonbj @TMan22

### CODE REVIEW

**Comment by Pilewski** [comment]
Thanks 💪

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: false
```
Comment (martensonbj) [inline app/models/reminder.js]: Let's change this to be a boolean with a default value - check out [the docs](https://guides.emberjs.com/v2.10.0/models/defining-models/#toc_options)

File: app/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel(/* transition */) {
+   this.transitionTo('reminders'); // Implicitly aborts the on-going transition.
```
Comment (martensonbj) [inline app/routes/index.js]: Comments should be removed

File: app/routes/reminders.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return this.get('store').findAll('reminder')
+  }
```
Comment (martensonbj) [inline app/routes/reminders.js]: Watch your indentation

File: app/templates/index.hbs [hunk 4]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (martensonbj) [inline app/templates/index.hbs]: If we're not using this file let's get rid of it

File: mirage/factories/reminder.js [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import { Factory, faker } from 'ember-cli-mirage';
 
 export default Factory.extend({
-  title: () => faker.lorem.words().join(' ').capitalize(),
+  title: () => faker.random.words(),
   date: () => faker.date.recent(3),
-  notes: () => faker.lorem.paragraph(),
+  notes: () => faker.hacker.phrase(),
```
Comment (martensonbj) [inline mirage/factories/reminder.js]: 👍

File: tests/unit/routes/index-test.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+moduleFor('route:index', 'Unit | Route | index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
+  let route = this.subject();
+  assert.ok(route);
+});
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: This test isn't necessary if you aren't using an index route.

**Comment by Pilewski** [comment]
@martensonbj added the changes you requested

**Comment by Pilewski** [comment]
This PR also Closes Issue #2

**Pull request merged**